src/app/tests/TestCommandInteraction.cpp

```diff
@@ -161,7 +161,7 @@ void DispatchSingleClusterCommand(const ConcreteCommandPath & aCommandPath, chip
         }
```

Comment:
Adding blocking comment:

Still need to add unit tests. For the time being this PR was uploaded in case there is a fundamental change that needs to be made. Figured I could get some initial reviews while I worked in parallel to add test, which will be in a separate file.